# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/layers/FrameMetrics.h
# Commit: 77ef8e8bc669
# Full Hash: 77ef8e8bc669c9ecab4d94505b11b3cb732446c5
# Author: Alexis Beingessner <a.beingessner@gmail.com>
# Date: 2019-07-15 21:39:46
# Regressor Bug: 1547351
# File Overlap Count: 2
# Description:
#   Bug 1547351 - Automatically set content process render root r=kats
#   
#   This implements the idea of automatically setting a content proc's
#   render root based on the render root enclosing the iframe that
#   points to it. There was a bit of cleanup in here that was a bit
# ==============================================================================

diff -r b594f8e47025 -r 77ef8e8bc669 gfx/layers/FrameMetrics.h
--- a/gfx/layers/FrameMetrics.h	Mon Jul 15 13:20:31 2019 +0000
+++ b/gfx/layers/FrameMetrics.h	Mon Jul 15 13:18:44 2019 +0000
@@ -12,6 +12,7 @@
 #include "Units.h"                  // for CSSRect, CSSPixel, etc
 #include "mozilla/DefineEnum.h"     // for MOZ_DEFINE_ENUM
 #include "mozilla/HashFunctions.h"  // for HashGeneric
+#include "mozilla/HashTable.h"      // for HashMap
 #include "mozilla/Maybe.h"
 #include "mozilla/gfx/BasePoint.h"               // for BasePoint
 #include "mozilla/gfx/Rect.h"                    // for RoundedIn
@@ -1099,7 +1100,7 @@
   // Please add new fields above this comment.
 };
 
-typedef std::map<ScrollableLayerGuid::ViewID, ScrollUpdateInfo>
+typedef HashMap<ScrollableLayerGuid::ViewID, ScrollUpdateInfo>
     ScrollUpdatesMap;
 
 }  // namespace layers